astronomer/providers/google/cloud/operators/bigquery.py

```diff
@@ -72,6 +72,10 @@ class BigQueryInsertJobOperatorAsync(BigQueryInsertJobOperator, BaseOperator):
     :param poll_interval: polling period in seconds to check for the status of job. Defaults to 4 seconds.
     """
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        self.poll_interval = kwargs.pop("poll_interval", None)
```

Comment:
```suggestion
        self.poll_interval: float = kwargs.pop("poll_interval", 4.0)
```
The default value seems to be 4.0 in the OSS provider. I guess we should have the default value here too and across other places in the PR.

Reply:
> Also, I guess whatever value the user gives for the `poll_interval`, it will be replaced by the super class `init` with the default value since we are calling init afterwards, no?
> 
> Instead of
> 
> ```
> self.poll_interval = kwargs.pop("poll_interval", None)
> super().__init__(*args, **kwargs)
> ```
> 
> Do we need to do something like this?
> 
> ```
> poll_interval: float = kwargs.pop("poll_interval", 4.0)
> super().__init__(*args, **kwargs)
> self.poll_interval = poll_interval
> ```
> 
> Attaching my trial in IPython <img alt="Screenshot 2023-04-19 at 10 17 58 PM" width="506" src="https://user-images.githubusercontent.com/10206082/233144853-4028546a-4e77-4de3-8279-c51badb5be70.png">

@pankajkoti 

```
super().__init__(*args, **kwargs)
self.poll_interval = poll_interval
```
^^ This will fail for the old provider version since it doesn't expect the `poll_interval` No?